### ISSUE TITLE
Busyloop when server offline

### ISSUE THREAD

**Issue by vranki** [vranki/hemppa#218]
It's been reported Hemppa hogs CPU when it can't reach homeserver. Check if there is a busy loop in this situation and fix.

**Comment by mijutu** [comment]
Yes, there is.  More than 50 sync attempts per second when answer is 503.
